README.md

```diff
@@ -15,8 +15,19 @@ You can then view the built documentation in the `_build` folder.
 
 ## Citation
 
-pyHyp is based on the theory presented in "Enhancements of a three-dimensional hyperbolic grid generation scheme."
-For more background, theory, and figures, see the original [journal article](https://doi.org/10.1016/0096-3003(92)90073-A).
+If you use pyHyp in any publication for which you find it useful, please cite this paper.
+
+N. Secco, G. K. W. Kenway, P. He, C. A. Mader, and J. R. R. A. Martins, “Efficient Mesh Generation and Deformation for Aerodynamic Shape Optimization”, AIAA Journal, 2021. [doi:10.2514/1.J059491](https://doi.org/10.2514/1.J059491)
+
+```
+@article{Secco2020a,
```

Comment:
This should probably be Secco2021a? I guess the bib file hasn't been update yet either.

Reply:
Good catch. Just changed it to Secco2021.